app/javascript/controllers/modal_controller.js

```diff
@@ -12,7 +12,6 @@ export default class ModalController extends Controller {
 
   close() {
     Promise.all(this.transitionableTargets.map((element) => leave(element))).then(() => {
-      this.element.parentElement.removeAttribute('src');
```

Comment:
This wasn't needed and causing some issues when re-rendering invalid form responses with the modal.